x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -46,3 +46,24 @@ export const CSP_RULE_TEMPLATE_SAVED_OBJECT_TYPE = 'csp-rule-template';
 
 export const CLOUDBEAT_VANILLA = 'cloudbeat/cis_k8s'; // Integration input
 export const CLOUDBEAT_EKS = 'cloudbeat/cis_eks'; // Integration input
+export const CLOUDBEAT_AWS = 'cloudbeat/cis_aws'; // Integration input
+export const CLOUDBEAT_GCP = 'cloudbeat/cis_gcp'; // Integration input
+export const CLOUDBEAT_AZURE = 'cloudbeat/cis_azure'; // Integration input
+export type CLOUDBEAT_INTEGRATION =
+  | typeof CLOUDBEAT_VANILLA
+  | typeof CLOUDBEAT_EKS
+  | typeof CLOUDBEAT_AWS
+  | typeof CLOUDBEAT_GCP
+  | typeof CLOUDBEAT_AZURE;
+export const KSPM_POLICY_TEMPLATE = 'kspm';
+export const CSPM_POLICY_TEMPLATE = 'cspm';
+export type POLICY_TEMPLATE = typeof KSPM_POLICY_TEMPLATE | typeof CSPM_POLICY_TEMPLATE;
+
+export const SUPPORTED_POLICY_TEMPLATES = [KSPM_POLICY_TEMPLATE, CSPM_POLICY_TEMPLATE];
+export const SUPPORTED_CLOUDBEAT_INPUTS = [
+  CLOUDBEAT_VANILLA,
+  CLOUDBEAT_EKS,
+  CLOUDBEAT_AWS,
+  CLOUDBEAT_GCP,
+  CLOUDBEAT_AZURE,
+];
```

Comment:
```suggestion
export const CLOUDBEAT_AWS = 'cloudbeat/cis_aws'; // Integration input
export const CLOUDBEAT_GCP = 'cloudbeat/cis_gcp'; // Integration input
export const CLOUDBEAT_AZURE = 'cloudbeat/cis_azure'; // Integration input

export const SUPPORTED_CLOUDBEAT_INPUTS = [
  CLOUDBEAT_VANILLA,
  CLOUDBEAT_EKS,
  CLOUDBEAT_AWS,
  CLOUDBEAT_GCP,
  CLOUDBEAT_AZURE,
];

// this will result in the same union type as you manually added 
export type CLOUDBEAT_INTEGRATION = typeof SUPPORTED_CLOUDBEAT_INPUTS[number];
 
export const KSPM_POLICY_TEMPLATE = 'kspm';
export const CSPM_POLICY_TEMPLATE = 'cspm';

export const SUPPORTED_POLICY_TEMPLATES = [KSPM_POLICY_TEMPLATE, CSPM_POLICY_TEMPLATE];

// same here 
export type POLICY_TEMPLATE = typeof SUPPORTED_POLICY_TEMPLATES[number];
```